twarc.py

```diff
@@ -140,7 +141,16 @@ def main():
         tweets = t.filter(track=args.track, follow=args.follow,
                           locations=args.locations)
     elif args.hydrate:
-        tweets = t.hydrate(open(args.hydrate, 'rU'))
+        f_names = [args.hydrate] if args.hydrate != '--hydrate' else None
```

Comment:
I wee. Well I've tested the use of '--hydrate -' with what is now on master and it works fine. Thanks very much for your contribution @dimazest